<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task 2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="figureTablayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Square</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rectangle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Triangle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Circle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rhombus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parallelogram</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hexagon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5-Star</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6-Star</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8-Star</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_figureTab">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Figure</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_figure">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="radiusLayout">
            <item>
             <widget class="QLabel" name="label_Radius">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Radius :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_radius">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="LWLayout">
            <item>
             <layout class="QHBoxLayout" name="lengthLayout">
              <item>
               <widget class="QLabel" name="label_Length">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Length :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_length">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="widthLayout">
              <item>
               <widget class="QLabel" name="label_Width">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Width :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_width">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_angle">
                <property name="text">
                 <string>Angle: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_angle">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>179</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="RadiusABLayout">
            <item>
             <layout class="QHBoxLayout" name="radiusALayout">
              <item>
               <widget class="QLabel" name="label_RadiusA">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>RadiusA :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_radiusA">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="radiusBLayout">
              <item>
               <widget class="QLabel" name="label_RadiusB">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>RadiusB :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_radiusB">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="sideLayout">
            <item>
             <widget class="QLabel" name="label_Side">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Side :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_side">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_5">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton_draw">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Draw</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_10"/>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget_right">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_6">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="interactive">
           <bool>false</bool>
          </property>
          <property name="sceneRect">
           <rectf>
            <x>0.000000000000000</x>
            <y>0.000000000000000</y>
            <width>1.000000000000000</width>
            <height>1.000000000000000</height>
           </rectf>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="transformationAnchor">
           <enum>QGraphicsView::AnchorViewCenter</enum>
          </property>
          <property name="resizeAnchor">
           <enum>QGraphicsView::AnchorViewCenter</enum>
          </property>
          <property name="viewportUpdateMode">
           <enum>QGraphicsView::FullViewportUpdate</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="positionLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_positionTab">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Position</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="positionLayout_2">
              <item>
               <widget class="QLabel" name="label_position">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Position : </string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="Position">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>(0;0)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_moveup">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_moveleft">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Left</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_moveright">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Right</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_movedown">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="originLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_originTab">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Origin</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="originPositionLayout">
              <item>
               <widget class="QLabel" name="label_origin">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Position : </string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="Origin">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>(0;0)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_origin_up">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_origin_left">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Left</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_origin_right">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Right</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_origin_down">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="miscLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_miscTab">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Misc</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_rotleft">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Rotate left</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_rotright">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Rotate right</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_sizeup">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Scale up</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_sizedown">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Scale down</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <layout class="QHBoxLayout" name="perimeterLayout">
              <item>
               <widget class="QLabel" name="label_perimeter">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Perimeter : </string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="Perimeter">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="6" column="0" colspan="2">
             <layout class="QHBoxLayout" name="areaLayout">
              <item>
               <widget class="QLabel" name="label_area">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Area : </string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="Area">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QPushButton" name="pushButton_reset">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
